### PR TITLE
ami: update verify of io.conf

### PR DESCRIPTION
**Pull request by amoskong**
The io.conf setup is changed from 2.3, the properties is moved to
/etc/scylla.d/io_properties.yaml

### CODE REVIEW

**Comment by amoskong** [comment]
Note: jenkins has problem, the PR isn't tested.

File: scylla-artifacts.py [hunk 1]
```diff
@@ -591,17 +591,17 @@ def enhanced_net_enabled(self):
...
-            'i3.2xlarge': ['SEASTAR_IO="--num-io-queues 8 --max-io-requests 192"', 'CPUSET="--cpuset 0-7 "'],
-            'i3.4xlarge': ['SEASTAR_IO="--num-io-queues 14 --max-io-requests 384"', 'CPUSET="--cpuset 1-7,9-15 "'],
-            'i3.8xlarge': ['SEASTAR_IO="--num-io-queues 30 --max-io-requests 768"', 'CPUSET="--cpuset 1-15,17-31 "'],
-            'i3.16xlarge': ['SEASTAR_IO="--num-io-queues 62 --max-io-requests 1536"', 'CPUSET="--cpuset 1-31,33-63 "'],
-            'c3.large': ['SEASTAR_IO="--num-io-queues 2 --max-io-requests 32"', 'CPUSET="--cpuset 0-1 "'],
-            'c3.8xlarge': ['SEASTAR_IO="--num-io-queues 8 --max-io-requests 32"', 'CPUSET="--cpuset 1-15,17-31 "'],
-            'm3.2xlarge': ['SEASTAR_IO="--num-io-queues 8 --max-io-requests 32"', 'CPUSET="--cpuset 0-7 "'],
-            'm3.medium': ['SEASTAR_IO="--num-io-queues 1 --max-io-requests 32"', 'CPUSET="--cpuset 0 "'],
-            'i2.4xlarge': ['SEASTAR_IO="--num-io-queues 14 --max-io-requests 128"', 'CPUSET="--cpuset 1-7,9-15 "']
+            'i3.large': ['SEASTAR_IO="--io-properties-file=/etc/scylla.d/io_properties.yaml"', 'CPUSET="--cpuset 0-1 "'],
```
[the remark this answers is not in the document]
Comment (roydahan) [reply scylla-artifacts.py]: I just saw you commented about it in Trello.

[the remark this answers is not in the document]
Comment (amoskong) [reply scylla-artifacts.py]: The io_properties.yaml isn't static, it's dynamically generated by scylla_io_setup script.

```
# cat /etc/scylla.d/io_properties.yaml
disks:
  - mountpoint: /var/lib/scylla/data
    read_iops: 752
    read_bandwidth: 127504168
    write_iops: 1505
    write_bandwidth: 74871176
```

related:  https://trello.com/c/SCSUpyzf/432-artifact-test-branch-23-update-verify-for-ioconf-setup#comment-5b56ea3a8e2850978dbdd518

[the remark this answers is not in the document]
Comment (roydahan) [reply scylla-artifacts.py]: It's static in terms of not generating a tool to detect performance.
It's calculated based on number of disks.
For now, it's ok.

**Comment by roydahan** [comment]
@amoskong, please backport this to branch-2.3 and make sure the jobs are running with correct branch.

**Comment by amoskong** [comment]
Backported one patch to branch-2.3:

[PATCH] ami: update verify of io.conf
(cherry picked from commit fcea630db417ac8ff681139f8b901da1340eee37)

I had confirmed all 2.3 artifact-test uses branch-2.3 of
scylla-artifact-tests. I will check again today.

**Comment by amoskong** [comment]
scylla-enterprise/ami-test failed for io.conf checking.
[job link ](http://jenkins.scylladb.com/job/scylla-enterprise/job/ami-test/label=muninn/3/artifact/job-2019-05-28T18.23-b74c207/job.log)

This PR needs to be backported to scylla-artifact-tests/branch-enterprise

**Comment by amoskong** [comment]
Backported this PR to branch-enterprise.

**Pull request merged**